test/instrumentation/main.go

```diff
@@ -35,6 +36,17 @@ const (
 	kubeMetricImportPath = `"k8s.io/component-base/metrics"`
 	// Should equal to final directory name of kubeMetricImportPath
 	kubeMetricsDefaultImportName = "metrics"
+
+	kubeURLRoot = "k8s.io/kubernetes"
+)
+
+var (
+	// env configs
+	GOROOT string = os.Getenv("GOROOT")
+	GOOS string = os.Getenv("GOOS")
+	KUBE_ROOT string = os.Getenv("KUBE_ROOT")
+
+	kubeRootDeSuffixed string = kubeRootDesuffix(KUBE_ROOT)
```

Comment:
converted to function so unit tests can optionally override the value of KUBE_ROOT and use the same package load process to set this value.